MaxText/layers.py

```diff
@@ -114,37 +124,16 @@ def dot_product_attention(query: Array,
   assert key.shape[-3] == value.shape[-3], 'k, v lengths must match.'
   assert query.shape[-1] == key.shape[-1], 'q, k depths must match.'
 
-  def compute_qk_attn_weights(query, key, cfg, aqt_rng):
+  def compute_qk_attn_weights(query, key):
     """Computes all query-key dot product pairs"""
-    if not cfg.int8_training:
-      attn_weights = jnp.einsum('bqhd,bkhd->bhqk', query, key)
-    else:
-      aqt_cfg = aqt_config.quantization_config(
-        cfg.fwd_int8_qk,
-        cfg.dlhs_int8_qk,
-        cfg.drhs_int8_qk,
-        use_fwd_quant=cfg.aqt_use_fwd_quant,
-        use_dummy_static_bound=cfg.aqt_use_dummy_static_bound,
-        rng_type=cfg.aqt_rng_type
-      )
-      aqt_dot_general = aqt.make_dot_general(aqt_cfg)
-      context = aqt.Context(key=aqt_rng, train_step=None)
-      aqt_dot_general = functools.partial(aqt_dot_general, context=context)
-      attn_weights = jnp.einsum('bqhd,bkhd->bhqk', query, key, _dot_general=aqt_dot_general)
-    return attn_weights
+    return jnp.einsum('bqhd,bkhd->bhqk', query, key)
```

Comment:
Are we abandonizing quantizing this matmul? Seems fine but GTK

Reply:
Yes, convergence issues for QK